pom.xml

```diff
@@ -154,67 +154,62 @@
       <groupId>com.vaadin</groupId>
     </dependency>
 
+    <!-- Added to provide logging output as Flow uses -->
```

Comment:
Don't use any mentions of NDA-protected applications please. Rework this comment to make it tenant-neutral.